### PR TITLE
Update steps to new designs

### PR DESCRIPTION
**Pull request by rvsia**
https://projects.engineering.redhat.com/browse/TPINVTRY-1001

- use spinner instead of progress
- add another button when success
- top margin for final steps

image
image

### CODE REVIEW

File: packages/sources/src/addSourceWizard/FinalWizard.js [hunk 1]
```diff
@@ -34,22 +33,18 @@ const FinalWizard = ({
                     successfulMessage={ successfulMessage }
                     hideSourcesButton={ hideSourcesButton }
                     returnButtonTitle={ returnButtonTitle }
-                    progressStep={progressStep}
-                    progressTexts={progressTexts}
+                    secondaryActions={<Button variant="link" onClick={ reset }><FormattedMessage id="wizard.addAnotherSource" defaultMessage="Add another source"/></Button>}
```
Comment (Hyperkid123) [inline packages/sources/src/addSourceWizard/FinalWizard.js:36]: I hope that you are excited to hear that the imperative `formatMessage` from `useIntl` has better performance than any of  the `Formatted` components :wink:

Comment (rvsia) [reply packages/sources/src/addSourceWizard/FinalWizard.js:36]: But using elements is cleaner and much faster to write. So I don't worry too much about it. :smile:

**Comment by rvsia** [comment]
@karelhala looks like the bot is stuck :/

**Comment by karelhala** [comment]
@rvsia yeah, my travis token breaks for some reason and refuses to run builds....

**Comment by karelhala** [comment]
https://travis-ci.org/github/RedHatInsights/frontend-components/builds/707753391

**Comment by beccaglass** [comment]
@rvsia looks great! Minor tweak - can we just change the body text from 'Your source has been successfully added.' to 'Your source was successfully added.' Thanks!

**Pull request merged**